x-pack/plugins/security_solution/common/format_errors.ts

```diff
@@ -26,4 +26,6 @@ export const formatErrors = (errors: t.Errors): string[] => {
       return `Invalid value "${value}" supplied to "${suppliedValue}"`;
     }
   });
+
+  return uniq(err);
```

Comment:
Optional:
https://youmightnotneed.com/lodash/#uniq

But you should be able to use Set and remove your usage of lodash if you want, doesn't bother me either way. Will still approve.

Reply:
Ooooo bookmarking that site. I like that. I'll update to use `Set`